frontend/src/views/camp/NavigationCamp.vue

```diff
@@ -13,6 +13,10 @@
     </logo>
 
     <v-toolbar-items>
+      <v-btn :to="campRoute(camp())" text>
```

Comment:
Ich würde in der Funktion `campRoute` nun eher `dashboard` als default Wert annehmen.

Oder? Da möchten wir ja landen, wenn wir von ausserhalb einfach auf das Lager navigieren